scripts/gh_matrix_builder.py

```diff
@@ -72,8 +78,22 @@ def build_apache_config(overrides):
   return base_config
 
 # always test debug build on latest pg11 and latest pg12
-m["include"].append(build_debug_config({"pg":"11.8"}))
-m["include"].append(build_debug_config({"pg":"12.3"}))
+m["include"].append(build_debug_config({"pg":PG11_LATEST}))
+m["include"].append(build_debug_config({"pg":PG12_LATEST}))
+
+macos_config = {
+  "pg": PG12_LATEST,
+  "os": "macos-latest",
```

Comment:
Maybe use explicit version here, like ubuntu? Otherwise things might break for us when they bump latest.